bindings.go

```diff
@@ -429,6 +466,37 @@ func (d Device) PowerUsage() (uint, error) {
 	return uint(n), errorString(r)
 }
 
+// PowerLimit retrieves the power management limit associated with this device.
+func (d Device) PowerLimit() (uint, error) {
+	if C.nvmlHandle == nil {
+		return 0, errLibraryNotLoaded
+	}
+	var n C.uint
+	r := C.nvmlDeviceGetPowerManagementLimit(d.dev, &n)
+	return uint(n), errorString(r)
+}
+
+// PowerLimitConstraints retrieves information about possible values of power management limits on this device.
+func (d Device) PowerLimitConstraints() (uint, uint, error) {
+	if C.nvmlHandle == nil {
+		return 0, 0, errLibraryNotLoaded
+	}
+	var min C.uint
+	var max C.uint
+	r := C.nvmlDeviceGetPowerManagementLimitConstraints(d.dev, &min, &max)
+	return uint(min), uint(max), errorString(r)
+}
+
+// Set new power limit of this device.
+func (d Device) SetPowerLimit(pl uint) error {
```

Comment:
Can you remove this method? I would like to keep this library to only read things and not change anything.